### PR TITLE
Import from template library guide

### PR DESCRIPTION
**Pull request by mclacore**

### CODE REVIEW

File: docs/getting_started/03-import-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+![Configure template](./img/library-configure-template.gif)
+
+Once you've configured the template, click `Deploy` to deploy the template to your cloud provider.
+
+## Next steps
+
+### Store your custom template in source control
+
+After you have customized and deployed your template, it is a good practice to store your template in source control (e.g. GitHub, Azure Repos, etc.). This will allow you to track changes to your template over time and collaborate with others.
```
Comment (chrisghill) [inline docs/getting_started/03-import-marketplace.md:57]: Add some details, and perhaps a gif on downloading bundle source code.

File: docs/getting_started/03-import-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+id: getting-started-import-template
+slug: /getting-started/import-template
+title: Import Template from Library
+sidebar_label: Import Template from Library
+---
+
+## Massdriver Template Library
+
+Running production-grade infrastructure in the cloud has never been easier with the Massdriver Template Library. The library is a collection of professionally curated infrastructure-as-code (IaC) templates that are designed to help you get started quickly with your cloud infrastructure. The templates are designed to be modular and can be easily customized to fit your specific needs.
```
Comment (coryodaniel) [inline docs/getting_started/03-import-marketplace.md:10]: Love this

**Review by coryodaniel — APPROVED** [review]
These are banging.

File: docs/getting_started/03-import-marketplace.md (hunk 1 above)
Comment (mclacore) [reply docs/getting_started/03-import-marketplace.md:57]: https://docs-git-michael-orc-574import-from-marketplace.preview.massdriver.cloud/getting-started/import-template#step-1-customize-the-template already in the doc

Comment (mclacore) [reply docs/getting_started/03-import-marketplace.md:57]: ill add a header link in this section though. i like that

**Review by chrisghill — APPROVED** [review]

**Pull request merged**